File: music-android-java/module_base/src/main/java/com/czb/module_base/utils/LogUtils.java
package com.czb.module_base.utils;

import android.util.Log;

public class LogUtils {
    //等级控制
    public static int currentLev = 5;

    public static final int DEBUG_LEV = 4;
    public static final int INFO_LEV = 3;
    public static final int WARNING_LEV = 2;
    public static final int ERROR_LEV = 1;

    public static void d(Object object,String log){
        if (currentLev> DEBUG_LEV){
            Log.d(object.getClass().getSimpleName(),log);
        }
    }
    public static void i(Object object,String log){
        if (currentLev> INFO_LEV){
            Log.d(object.getClass().getSimpleName(),log);
        }
    }

    public static void w(Object object,String log){
        if (currentLev> WARNING_LEV){
            Log.d(object.getClass().getSimpleName(),log);
        }
    }

    public static void e(Object object,String log){
        if (currentLev> ERROR_LEV){
            Log.d(object.getClass().getSimpleName(),log);
        }
    }

}

File: music-android/module_base/src/main/java/com/chen/module_base/utils/LogUtils.kt
package com.chen.module_base.utils

import android.util.Log
import com.chen.module_base.BuildConfig


object LogUtils {
    private const val TAG = "com.chan.module_base"

    private var DEBUG = BuildConfig.DEBUG

    fun d(tag:String = TAG, msg: String) {
        if (DEBUG) {
            Log.d(tag, msg)
        }
    }

    fun e(tag:String = TAG,msg: String) {
        if (DEBUG) {
            Log.e(tag, msg)
        }
    }

    fun i(tag:String = TAG,msg: String) {
        if (DEBUG) {
            Log.i(tag, msg)
        }
    }

    fun v(tag:String = TAG,msg: String) {
        if (DEBUG) {
            Log.v(tag, msg)
        }
    }

    fun w(tag:String = TAG,msg: String) {
        if (DEBUG) {
            Log.w(tag, msg)
        }
    }
}